ann_benchmarks/main.py

```diff
@@ -130,6 +169,18 @@ def main():
     if args.list_algorithms:
         list_algorithms(args.definitions)
         sys.exit(0)
+    
+    if args.build_only and args.test_only:
+        raise Exception('Nothing to run (build only and test only was specified)')
+    if (args.build_only or args.test_only) and not args.local:
+        raise Exception('Can\'t run build or test only on docker')
+
+    if args.user and not args.auth:
+        raise Exception('username specified without password (Auth)')
+    conn_params = {'host': args.host, 'port': args.port, 'auth': args.auth, 'user': args.user}
```

Comment:
this is redisearch problem, no?

Reply:
Yes, but it easy to test. Can be removed